### DISCUSSION TITLE
doris 外部引擎用的es，在doris查询中如何返回es 中 array类型字段的所有值

### DISCUSSION THREAD

**Discussion by WesleyZiwen** [apache/doris#18347] [A - General / Q&A]
doris 表中的 字段是 custom_tags（bigint） 对应es中的字段是  "custom_tags" : [
            223130227184017,
            223330227299044
          ],

如何查询的时候拿到  array里面的 2个值

**Comment by yagagagaga** [comment]
Do you use Doris ES catalog? What's your Doris version? I need more information to help you.
